docs/reference/getting-started.asciidoc

```diff
@@ -788,7 +788,7 @@ As for the response, we see the following parts:
 * `hits` – search results
 * `hits.total` – total number of documents matching our search criteria
 * `hits.hits` – actual array of search results (defaults to first 10 documents)
-* `hits.sort` - sort key for results (missing if sorting by score)
+* `hits.sort` - sort value for results (missing if sorting by score)
```

Comment:
if you really want to be fussy, it could probably be "value of the sort key for each result"

Reply:
I have made the changes.